e2e/cephfs.go

```diff
@@ -768,7 +768,7 @@ var _ = Describe(cephfsType, func() {
 				for i := range deplPods {
 					err = ensureStatSucceeds(deplPods[i].Name)
 					if err != nil {
-						framework.Failf(err.Error())
+						framework.Failf("%v", err.Error())
```

Comment:
a bit more context in the errors would help here.

```go
framework.Failf("ensureStatSucceeds failed for pod %q: %v", deplPods[i].Name, err.Error())
```

Something like that really helps the person receiving the error a lot. These things can/should be done everywhere.